README.pt.md

```diff
@@ -0,0 +1,121 @@
+|       19       |                    Hotéis romanticos da Europa ♥️                     | [Processamento de linguagem natural](6-NLP/README.md) | Análise de sentimento com avaliações de hotéis, 1                                                                                    |      [aula](6-NLP/4-Hotel-Reviews-1/README.md)      |   Stephen    |
+|       20       |                    Hotéis romanticos da Europa ♥️                     | [Processamento de linguagem natural](6-NLP/README.md) | Análise de sentimento com avaliações de hotéis, 2                                                                                    |      [aula](6-NLP/5-Hotel-Reviews-2/README.md)      |   Stephen    |
+|       21       |                Introdução previsão de séries temporais                |      [Séries temporais](7-TimeSeries/README.md)       | Introdução a previsão de séries temporais                                                                                            |    [aula](7-TimeSeries/1-Introduction/README.md)    |  Francesca   |
+|       22       | ⚡️ Uso de Energia Mundial ⚡️ - previsão de série temporal com ARIMA |      [Séries temporais](7-TimeSeries/README.md)       | Previsão de séries temporais com ARIMA                                                                                               |       [aula](7-TimeSeries/2-ARIMA/README.md)        |  Francesca   |
+|       23       |                 Introdução à aprendizagem por reforço                 | [Aprendizagem por reforço](8-Reinforcement/README.md) | Introdução à aprendizagem por reforço com Q-Learning                                                                                 |    [aula](8-Reinforcement/1-QLearning/README.md)    |    Dmitry    |
+|       24       |                  Ajuda o Pedro a escapar do lobo! 🐺                  | [Aprendizagem por reforço](8-Reinforcement/README.md) | Ginásio de aprendizagem por reforço                                                                                                  |       [aula](8-Reinforcement/2-Gym/README.md)       |    Dmitry    |
+|   Postscript   |                   Cenários reais de ML e aplicações                   |       [ML na vida real](9-Real-World/README.md)       | Aplicações interessantes e reveladoras do mundo real do ML clássico                                                                  |    [aula](9-Real-World/1-Applications/README.md)    |    Equipa    |
+
+## Acesso Offline
+
+Podes usar este documento offline usando o [Docsify](https://docsify.js.org/#/). Dá fork neste repositório, [instala o Docsify](https://docsify.js.org/#/quickstart) no teu computador, e depois na pasta raiz deste repositório, escreve `docsify serve`. O site será executado na porta 3000 do teu localhost: `localhost:3000`.
+
+## PDFs
+
+Encontras um pdf do curso com links [aqui](pdf/readme.pdf)
+
+## Help Wanted!
```

Comment:
Do you think this should also get translated?

Reply:
Will do it 😀